### ISSUE TITLE
[enhancement]: could users be allowed to use sdxl for inpainting?

### ISSUE THREAD

**Issue by than0107** [invoke-ai/InvokeAI#4337]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What should this feature add?

I would like to use sdxl for inpainting, but I cannot find sdxl in inpainting (I can find it in text2prompt) probably because there is no inpainting variant of sdxl. I googled and found this tread in which some user posted that there could be no inpainting sdxl in the future as stabilityai doesn't deem it necessary. https://www.reddit.com/r/StableDiffusion/comments/15eawt6/sdxl_inpainting/

So in short, could invokeai allow users to use sdxl for inpainting?

### Alternatives

I can do sdxl inpainting with automatic1111 though. 
No this is not an alternative, I want invokeai.

**Comment by 00xBAD** [comment]
I have tried to use this:
https://huggingface.co/diffusers/controlnet-canny-sdxl-1.0

But even though I put in the correct sdxl/controlnet folder, it doesn't show up in the UI.

But if I change the model, like sd-1 or sd-2, it shows up on the controlnet panels, always as sd-2 controlnet, no matter what...

**Comment by Millu** [comment]
This will be coming in 3.1!
